### DISCUSSION TITLE
Only sound, No video - Only video, no sound

### DISCUSSION THREAD

**Discussion by slussinator** [olivierfriard/BORIS#462] [Q&A]
When playing an observation, at first I hear only the sound. When I push the film-symbol the video appears, but the sound stops. I am using the newest version of BORIS and am using VLC compatible video files. 

Edit: I am using boris-7.12.2 for macOS on macOS Monterey version 12.1

Any ideas how to make it work?

Thanks in advance!

**Comment by olivierfriard** [comment]
Hi Sophie,

please see http://www.boris.unito.it/pages/report_a_bug.html

Best regards



On Tue, Jan 18, 2022 at 12:25 PM sophielusser ***@***.***>
wrote:

> When playing an observation, at first I hear only the sound. When I push
> the film-symbol the video appears, but the sound stops. I am using the
> newest version of BORIS and am using VLC compatible video files.
>
> Any ideas how to make it work?
>
> Thanks in advance!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/olivierfriard/BORIS/discussions/462>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABR2AYRIQJENA3T64R3OWPTUWVE2JANCNFSM5MG5CEGQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
University of Torino
Dept of Life Sciences and Systems Biology
Via dell'Accademia Albertina, 13 10123 TORINO (Italy)
tel: +39 011 6704542
http://penelope.unito.it/friard/
GitHub: olivierfriard <https://github.com/olivierfriard>
ORCID: 0000-0002-0374-9872 <http://orcid.org/0000-0002-0374-9872>
Web of Science ResearcherID: ABG-2435-2021
<https://publons.com/researcher/4857666/olivier-friard/>
Scopus Author ID: 10639803800
<https://www.scopus.com/authid/detail.uri?authorId=10639803800>

**Comment by javarhol** [comment]
I was having a similar problem and when I hit the jump forward button the video appears and operates normally.
